packages/story-editor/src/utils/getBlurHashFromImage.js

```diff
@@ -37,7 +37,12 @@ const getImageData = (image) => {
 };
 
 const getBlurHashFromImage = async (src) => {
-  const image = await preloadImage(src);
+  let image;
+  try {
+    image = await preloadImage(src);
+  } catch {
+    // Ignore
```

Comment:
We should not ignore this. If the image fails to load here, we should error out or something. 

```suggestion
    return Promise.reject(new Error('image failed to load'));
```